pyfilesync/console.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Program makes backup of your file
+Example usage
+=============
+
+To invoke program type in your console::
+
+    filesync --help
+
+"""
+
+import logging
+from argparse import ArgumentParser
+from pyfilesync import get_version
+
+
+def __option_args(argv=None):
+    """
+    Parsing argument for command line program.
+
+    @param argv: Option parameters
+    @type argv: list
+    @return: parsed arguments
+    """
+    parser = ArgumentParser(description="Make backup of your files.",
+                            prog='filesync')
+    parser.add_argument("-v", "--version", action='version',
+                        version='%(prog)s {}'.format(get_version()))
+    parser.add_argument("-l", "--log", dest="logLevel",
+                        choices=['DEBUG', 'INFO', 'WARNING',
+                                 'ERROR', 'CRITICAL'],
+                        help="Set the logging level")
+    try:
+        argcomplete.autocomplete(parser)
+    except NameError:
+        pass
+    return parser.parse_args(argv)
```

Comment:
Tutaj jest samo parsowanie. Jeśli podamy nulla to zostaną pobrane parametry programu konsolowego